.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
-  "editor.formatOnSave": true,
-  "[javascript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-  },
-  "prettier.configPath": ".prettierrc"
+    "editor.formatOnSave": true,
+    "[javascript]": {
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
+    },
+    "prettier.configPath": ".prettierrc"
```

Comment:
I'll have to add the config for JSON files, because your shitty computer keeps adding 4 spaces.

Reply:
I already added it on another PR. So, don't fix it here.